LiskHQ/lisk
# Modify validation chain and update config.json - Closes #2959

### What was the problem?
Review and update validation for `chain` component.

### How did I fix it?
Changes:
- `wsPort` needs to be an integer, however, few `/config/{network)/config.json` configurations still had a string (while some others had the correct integer).
- Added extra validation (see changes commit)

### How to test it?
Start network and log chain config to see how the default config is applied for each network type.

### Review checklist

* The PR resolves #2959
* All new code is covered with unit tests
* All new code was formatted with Prettier
* Linting passes
* Tests pass
* Commit messages follow the [commit guidelines](CONTRIBUTING.md#git-commit-messages)
* Documentation has been added/updated
